src/auth/SignUp.jsx

```diff
@@ -130,6 +134,7 @@ export const SignUp = () => {
               {error?.passwordEquality && (
                 <ErrorMessage>Passwords do not match</ErrorMessage>
               )}
+              {errorApi && <ErrorMessage>{errorApi}</ErrorMessage>}
```

Comment:
errorapi?

Reply:
That is going to be our personalized error, we already have an `error`variable in the component, the one for the validation on the client.

Reply:
maybe you can explain me tomorrow, i could not fin errorapi anywhere else

Reply:
I can explain to you, of course. But I am just wondering, did you used ctrl+f to search in the document? Did you use the magnifying class from VSCode to search in the whole code?

Reply:
ahhh there on top. i did not saw it there, lokked at the imports and all in the client file local if it already has been declared somewhere. sorry its late